global-script.lua

```diff
@@ -1900,125 +1900,133 @@ function resetPiece(object, rotation, depth)
+            Vector(14.64, 0.00, -5.90),
         },
     },
     { -- 4 Player
         { -- Standard
-            Vector{2.36, 1.08, 26.47},
-            Vector{20.40, 1.08, 26.64},
-            Vector{-6.65, 1.08, 11.13},
-            Vector{11.27, 1.08, 11.33},
+            Vector(-3.60, 0.00, 9.88),
+            Vector(14.44, 0.00, 10.05),
+            Vector(-12.61, 0.00, -5.46),
+            Vector(5.31, 0.00, -5.26),
         },
         { -- Leaf
-            Vector{7.05, 1.08, 34.30},
-            Vector{20.53, 1.08, 26.36},
-            Vector{-2.00, 1.08, 18.53},
-            Vector{11.39, 1.08, 10.92},
+            Vector(1.09, 0.00, 17.71),
+            Vector(14.57, 0.00, 9.77),
+            Vector(-7.96, 0.00, 1.94),
+            Vector(5.43, 0.00, -5.67),
         },
-        { --Snake
-            Vector{35.36, 1.08, 37.55},
-            Vector{8.26, 1.08, 22.19},
-            Vector{26.45, 1.08, 22.36},
-            Vector{-0.73, 1.08, 7.00},
+        { -- Snake
+            Vector(29.40, 0.00, 20.96),
+            Vector(2.30, 0.00, 5.60),
+            Vector(20.49, 0.00, 5.77),
+            Vector(-6.69, 0.00, -9.59),
         },
     },
     { -- 5 Player
         { -- Standard
-            Vector{3.32, 1.08, 32.42},
-            Vector{25.46, 1.08, 24.68},
-            Vector{38.99, 1.08, 32.44},
-            Vector{12.18, 1.08, 16.81},
-            Vector{25.62, 1.08, 9.32},
+            Vector(-2.64, 0.00, 15.83),
+            Vector(19.50, 0.00, 8.09),
+            Vector(33.03, 0.00, 15.85),
+            Vector(6.22, 0.00, 0.22),
+            Vector(19.66, 0.00, -7.27),
         },
-        {  -- Snail
-            Vector{26.42, 1.08, 41.16},
-            Vector{13.22, 1.08, 33.29},
-            Vector{26.68, 1.08, 25.70},
-            Vector{8.72, 1.08, 10.08},
-            Vector{26.67, 1.08, 9.98},
+        { -- Snail
+            Vector(20.46, 0.00, 24.57),
+            Vector(7.26, 0.00, 16.70),
+            Vector(20.72, 0.00, 9.11),
+            Vector(2.76, 0.00, -6.51),
+            Vector(20.71, 0.00, -6.61),
         },
         { -- Peninsula
-            Vector{10.81, 1.08, 32.03},
-            Vector{26.27, 1.08, 32.27},
-            Vector{18.66, 1.08, 18.81},
-            Vector{41.71, 1.07, 23.07},
-            Vector{57.12, 1.07, 13.96},
+            Vector(4.85, 0.00, 15.44),
+            Vector(20.31, 0.00, 15.68),
+            Vector(12.70, 0.00, 2.22),
+            Vector(35.75, 0.00, 6.48),
+            Vector(51.16, 0.00, -2.63),
         },
         { -- V
-            Vector{0.17, 1.07, 33.75},
-            Vector{40.67, 1.07, 41.60},
-            Vector{8.96, 1.07, 18.16},
-            Vector{31.52, 1.07, 26.14},
-            Vector{22.40, 1.07, 10.67},
+            Vector(-5.79, 0.00, 17.16),
+            Vector(34.71, 0.00, 25.01),
+            Vector(3.00, 0.00, 1.57),
+            Vector(25.56, 0.00, 9.55),
+            Vector(16.44, 0.00, -5.92),
         },
     },
     { -- 6 Player
```

Comment:
does the table still actually fit all of these super large maps?

Reply:
Just barely, which is why I shifted the star and leaf slightly. It has been a while since I wrote this, but I suspect the scaling factor was picked partly to make them fit.